### DISCUSSION TITLE
Single button Clip Record in Grid Mode

### DISCUSSION THREAD

**Discussion by WasKa001** [SynthstromAudible/DelugeFirmware#2824] [Ideas]
In grid mode, is it possible to arm and record to a new clip without pressing Record button? (Clip Arm Record always On)

Currently if I want to record to a clip with MIDI keyboard, I have to press Record button and then clip button. To stop recording, I have to press Record or Clip button again (no issue with that). However; I’d like to be able to only press the clip button and Record would be then engaged automatically, in a similar fashion as with Record Arm in Ableton. I don’t think this is currently available, but perhaps can be implement in further firmware releases?

**Comment by duncanrmi** [comment]
I think this would make sense if you were going into an empty clip each time, yes.... I think the extra step will save you going into record on the wrong clip if you make a mistake though.
